mlfinlab/data_structures/run_data_structures.py

```diff
@@ -171,18 +185,19 @@ def _update_cache(self, date_time, price, low_price, high_price, cum_theta_sell,
         :param cum_ticks: Cumulative number of ticks
         """
         cache_data = self.cache_tuple(date_time=date_time, price=price, high=high_price, low=low_price,
-                                      cum_ticks=cum_ticks, cum_volume=cum_volume, cum_theta_buy=cum_theta_buy,
+                                      cum_ticks=cum_ticks, buy_ticks=buy_ticks, cum_volume=cum_volume,
+                                      cum_theta_buy=cum_theta_buy,
                                       cum_theta_sell=cum_theta_sell)
         self.cache.append(cache_data)
 
     def _get_expected_imbalance(self, window, imbalance_array):
```

Comment:
get expected imbalance or get expected runs?

Reply:
Actually, we get expected imbalance. In run bars we calculate the same imbalance, the only difference is that we split buy and sell imbalance